src/main/java/seedu/address/model/person/Interaction.java

```diff
@@ -59,6 +78,10 @@ public String getOutcome() {
         return this.outcome.toString();
     }
 
+    public Date getDate() {
+        return this.date;
+    }
+
     @Override
     public String toString() {
         return interactionNote + "\nThe outcome of this interaction is: " + outcome;
```

Comment:
Shall we add the date to the string representation as well?

Reply:
Done!